package.json

```diff
@@ -5,9 +5,11 @@
   "description": "actual syncing server",
   "main": "index.js",
   "scripts": {
-    "start": "node app",
+    "start": "node build/app",
+    "dev": "ts-node app",
     "lint": "eslint .",
     "build": "tsc",
+    "clean": "rm -rf build",
```

Comment:
@jlongster adding a platform specific `rm -rf` to a script is not ideal. nothing else seems concerning atm.

Reply:
I think this is OK. We already assume an environment like git bash on windows. thanks!